modin/pandas/test/test_groupby.py

```diff
@@ -1145,11 +1221,15 @@ def eval_shift(modin_groupby, pandas_groupby):
         # Verify that there is no exceptions on our side
         repr(modin_groupby.shift(axis=1, fill_value=777))
     else:
-        eval_general(
-            modin_groupby,
-            pandas_groupby,
-            lambda groupby: groupby.shift(axis=1, fill_value=777),
-        )
+        # Modin does not raise an exception when trying to call any function
```

Comment:
Please, create an issue to look at the case.

Reply:
was able to fix this in 0c2f4a1bb76b48c3092a8372cc877ba4ee9416b9, so an issue tracker is not needed anymore